managed_vms/endpoints/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,13 @@
+runtime: java
```

Comment:
You don't need one for app.yaml files. And there isn't one for the hello world. Can you make that consistent across this repo?